helix-core/src/comment.rs

```diff
@@ -94,6 +96,24 @@ pub fn toggle_line_comments(doc: &Rope, selection: &Selection, token: Option<&st
     Transaction::change(doc, changes.into_iter())
 }
 
+/// Return token if the current line is commented.
+/// Otherwise, return None.
+pub fn continue_comment<'a>(doc: &Rope, line: usize, token: Option<&'a str>) -> Option<&'a str> {
```

Comment:
It makes more sense for this to return a bool since we already have the token in the calling code and `continue_comment` doesn't change it in any way.

Reply:
That would require to `unwrap()` the token, though. Do you still want me to do this change?